### ISSUE TITLE
Source map is not included in the package but is referenced by the min file

### ISSUE THREAD

**Issue by marcinbot** [godaddy-wordpress/wc-plugin-framework#360]
See https://wordpress.org/support/topic/file-missing-from-plugin-2/ - Safari logs the missing source map as an error to the dev console.

The map files should either be included in this package, or the JS should be compiled without source maps.

**Comment by retroneo** [comment]
This is still a problem in 2.1.1 and Safari logs are littered with these errors:

[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (sv-wc-payment-gateway-payment-form.min.js.map, line 0)

**Comment by jeffpaul** [comment]
We have a downstream extension merchant reporting a console on every page load due to an issue in the `sv-wc-payment-gateway-payment-form.min.css.map` file. The lack of space after `/*` seems to be what triggers the error.

Their suggestion to fix this is to change the following:

`/*# sourceMappingURL=sv-wc-payment-gateway-payment-form.min.css.map */`

to

`/* # sourceMappingURL=sv-wc-payment-gateway-payment-form.min.css.map */`

### Expected Behavior

No console errors coming from the said source map.

### Actual Behavior

Following error in console:

```.../wp-content/plugins/woocommerce-gateway-paypal-powered-by-braintree/vendor/skyverge/wc-plugin-framework/woocommerce/payment-gateway/assets/css/frontend/sv-wc-payment-gateway-payment-form.min.css.map Failed to load resource: the server responded with a status of 404 (Not Found)```

### Additional Details

See forum thread [here](https://wordpress.org/support/topic/sv-wc-payment-gateway-payment-form-min-css-map-error/#post-16943163).

**Comment by jeffpaul** [comment]
See related file location of concern: 
https://github.com/godaddy-wordpress/wc-plugin-framework/blob/c78046e2a97463afbc64a2b847dedfdc87bf38d7/woocommerce/payment-gateway/assets/css/frontend/sv-wc-payment-gateway-payment-form.min.css#L2

Noting that some of the other `min.css` files appear to have the same issue with no space before the `#` character so there might be a couple places to update if that is in fact causing the issue here.

**Comment by unfulvio-godaddy** [comment]
hey there thanks for logging this and apologies for the late reply

I have opened a PR and updated the build dependencies including `grunt-contrib-sass` hoping it would fix the issue or provide a config option to address it, but turns out there's no output change. I have manually edited the css files with the sourcemap ref but obviously this isn't a long term solution.

In the meantime I have also opened an issue in https://github.com/gruntjs/grunt-contrib-sass/issues/283 hoping that the package would address this.
